routes/orderRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+const orderController = require('../controllers/orderController');
+const { validateToken } = require('../middlewares/auth');
+
+router.post('/payment', validateToken, orderController.createOrder);
+
+module.exports = router;
```

Comment:
```suggestion
const express = require('express');

const orderController = require('../controllers/orderController');
const { validateToken } = require('../middlewares/auth');

const router = express.Router();

router.post('/payment', validateToken, orderController.createOrder);

module.exports = router;
```
외부 묘듈, 커스텀 모듈 순으로 띄어 쓰기를 통해 구분하며 불러오고, 이를 활용하는 변수를 선언하는 코드, 순으로 코드가 작성되는 것이 관례입니다~! 예시코드 참고해주세요!